commands/create_user

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+
+help() {
+  cat <<EOF
+Usage: $(basename "$0") [--username <username>] [--password <password>] [--email <email>]
+Ex: c8y oee \create_user --username abctest --password Password.123 --email dummy-oeeadmin@softwareag.com
+Create a user with global roles "OEE Admin" and "OEE Reader".
+EOF
+}
+
+echo "URL       :  $C8Y_HOST"
+echo "Tenant ID :  $C8Y_TENANT"
+echo "User      :  $C8Y_USER"
+
+# Set default values
+username="oeeadmin"
+password="Password.123"
+email="dummy-oeeadmin@softwareag.com"
+
+# Parse command-line options
+REST_ARGS=()
+while [ $# -gt 0 ]; do
+  case "$1" in
+    -h|--help)
+      help
+      exit 0
+      ;;
+    --username)
+      if [ $# -gt 1 ]; then
+        username=$2
+        shift
+      else
+        echo "Option $1 requires an argument." >&2
+        exit 1
+      fi
+      ;;
+    --password)
+      if [ $# -gt 1 ]; then
+        password=$2
+        shift
+      else
+        echo "Option $1 requires an argument." >&2
+        exit 1
+      fi
+      ;;
+    --email)
+      if [ $# -gt 1 ]; then
+        email=$2
+        shift
+      else
+        echo "Option $1 requires an argument." >&2
+        exit 1
+      fi
+      ;;
+    *)
+      REST_ARGS+=("$1")
+      ;;
+  esac
+  shift
+done
+set -- "${REST_ARGS[@]}"
+
+oeeadminUser=$(c8y users list --includeAll --username "$username" -o csv --select "id" -n)
+
+if [ -z "$oeeadminUser" ]; then
+  echo "$username user was not detected, creating $username user."
+
+  echo "Create user $username and add to OEE Admin, OEE Reader, business groups"
+  c8y users create -f -n --userName "$username" --password "$password" --email "$email" | \
```

Comment:
- please only show the complete response if there was an error (e.g. invalid password) and only show a short success message otherwise